.travis.yml

```diff
@@ -20,14 +20,14 @@ go_import_path: firebase.google.com/go
 
 before_install:
     # Golint requires Go 1.7 or later.
-    - if ! [[ "$TRAVIS_GO_VERSION" =~ ^1\.6\.([0-9]+|x)$ ]]; then go get github.com/golang/lint/golint; fi
+    - if ! [[ "$TRAVIS_GO_VERSION" =~ ^1\.6\.([0-9]+|x)$ || "$TRAVIS_GO_VERSION" =~ ^1\.7\.([0-9]+|x)$ || "$TRAVIS_GO_VERSION" =~ ^1\.8\.([0-9]+|x)$ ]]; then go get golang.org/x/lint/golint; fi
```

Comment:
cloud.google.com/go has stopped supporting go 1.8 and earlier, which means we can stop supporting them too. Can you drop those targets from the Travis config, and update this file accordingly?

Reply:
OK, now that we have removed support 1.6. Maybe we can update context Import `golang.org/x/netcontext` -> `context` and other tricks for 1.6 in another PR.

Reply:
Yes, lets make those changes in a separate PR.